templates/buildkite-elastic.yml

```diff
@@ -237,6 +237,14 @@ Parameters:
       - false
     Default: "true"
 
+  EnableDockerUserNamespaceRemap:
+    Type: String
+    Description: Enables experimental feature to run docker as buildkite-agent
```

Comment:
What if we let this feature be turned on using an ENV var?

If we do add it as another option on the stack setup screen, might need to make it clear that it's not "buildkite-agent" the binary we're taking about here in the description?

If it's a "this could be buggy, so you better know what you're doing" then perhaps we just say:

> Enables experimental Docker userns-remap support

Reply:
The issue is that it needs to have the system docker daemon started with the setting

Reply:
Ah of course! 🙃